.travis.yml

```diff
@@ -1,2 +1,8 @@
 language: node_js
 node_js: 6.2.2
+notifications:
+  email: false
+before_script:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+test: npm test --silent
```

Comment:
There's no `test` property mentioned in Travis docs. Do you mean this to be `script`? The default value for `script` in `language: node_js` is `npm test`, which I think is better than silent.

Also please use array notation so other scripts can be added

```yaml
script:
    - npm test
```